vc_zoom/indico_vc_zoom/task.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2020 - 2022 CERN and ENEA
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from requests.exceptions import HTTPError
+from sqlalchemy.orm.attributes import flag_modified
+
+from indico.core.celery import celery
+from indico.core.db import db
+
+
+def update_state_log(log_entry, failed):
+    log_entry.data['State'] = 'failed' if failed else 'succeeded'
+    flag_modified(log_entry, 'data')
+
+
+@celery.task(plugin='vc_zoom', max_retries=None)
+def refresh_meetings(vc_rooms, target_dt, log_entry=None):
+    from indico_vc_zoom.api import ZoomIndicoClient
+    client = ZoomIndicoClient()
+    failed = False
+    try:
+        for vc_room in [room for room in vc_rooms if room.type == 'zoom']:
```

Comment:
You already filter before calling the desk so no need to check again

Reply:
Yes, just trying to ensure some consistency.